packages/protocol/contracts/common/Signatures.sol

```diff
@@ -29,8 +31,14 @@ library Signatures {
     pure
     returns (address)
   {
-    bytes memory prefix = "\x19Ethereum Signed Message:\n32";
-    bytes32 prefixedHash = keccak256(abi.encodePacked(prefix, messageHash));
-    return ecrecover(prefixedHash, v, r, s);
+    uint256 v32 = v;
+    bytes memory signature = new bytes(96);
+    assembly {
```

Comment:
This recombination of arguments is unfortunate.  Should I send a PR to openzeppelin https://github.com/OpenZeppelin/openzeppelin-contracts/compare/master...jfoutts-celo:master ?

Reply:
@jfoutts-celo I think our teammates that maintain the contracts would appreciate it :D At least we should discuss if we should support this alternate way of calling it.